.vortex/tooling/src/helpers.php

```
@@ -1185,6 +1163,81 @@ function copy_dir(string $src, string $dst): void {
+function remove_dir(string $dir): void {
+  if (is_link($dir)) {
+    @unlink($dir);
+
+    return;
+  }
+
+  if (!is_dir($dir)) {
+    return;
+  }
+
+  $items = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($dir, \RecursiveDirectoryIterator::SKIP_DOTS), \RecursiveIteratorIterator::CHILD_FIRST);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '1222', column '16'). (undefined)

(MissingImport)